docs/4.1. Node to Authorization Server Interactions.md

```diff
@@ -227,4 +227,4 @@ Content-Type: application/json
 ]
 ```
 
-Note that if a Node is unable to contact an Authorization Server, it should attempt to contact another Authorization Server from the discovered list until this [either succeeds or the list is exhausted](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.5._Behaviour_-_Resource_Servers.html#public-keys).
+Note that if a Node is unable to contact an Authorization Server, it ought to attempt to contact another Authorization Server from the discovered list until this [either succeeds or the list is exhausted](https://specs.amwa.tv/is-10/branches/v1.0-dev/docs/4.5._Behaviour_-_Resource_Servers.html#public-keys).
```

Comment:
Did this ought to be made consistent with the other PR, Jonathan?

Reply:
Yes, you're right

Reply:
Rolled back to avoid PR conflict